stumpy/core.py

```diff
@@ -3343,3 +3529,36 @@ def _client_to_func(client):
     func = getattr(module, prefix + calling_func)
 
     return func
+
+
+def find_incompatible_args(func, required_args):
```

Comment:
Maybe make this a private function?

Reply:
Right! I think this should be private. I will revise it. Thank you for the suggestion!